app/policies/repository_policy.rb

```diff
@@ -1,4 +1,15 @@
 class RepositoryPolicy
+  attr_reader :user, :repository
```

Comment:
Do you mean the whole file or the line 2? If I remove the line, the test will fail with the following error message:

NameError:
       undefined local variable or method `user' for #<RepositoryPolicy:0x00000008f9b660>